CHANGELOG.md

```diff
@@ -119,6 +119,7 @@
    * FIXED: Reduce verbose instructions by collapsing small end ramp forks [#2762](https://github.com/valhalla/valhalla/issues/2762)
 
 * **Enhancement**
+   * ADDED: Add option to use thread-safe GraphTile's reference counter. [#2772](https://github.com/valhalla/valhalla/pull/2772)
```

Comment:
please put this at the bottom of the list

Reply:
Done.